google/cloud/sqlalchemy_spanner/requirements.py

```diff
@@ -68,3 +68,25 @@ def get_isolation_levels(self, _):
             dict: isolation levels description.
         """
         return {"default": "SERIALIZABLE", "supported": ["SERIALIZABLE", "AUTOCOMMIT"]}
+
+    @property
+    def precision_numerics_many_significant_digits(self):
+        """target backend supports values with many digits on both sides,
+        such as 319438950232418390.273596, 87673.594069654243
+
+        """
+        return exclusions.open()
+
+    @property
+    def precision_numerics_retains_significant_digits(self):
```

Comment:
Based on this override reason:
```
SPANNER OVERRIDE:
        Cloud Spanner ignores the `0`s after the decimal point if all
        the digits are `0`.
        Overriding the test to avoid the assertion failure.
```

We should not have this property open as it enables a single test that we have to override.